action.yml

```diff
@@ -0,0 +1,265 @@
+name: 'Build and Push Container Images'
+description: 'Builds images on PRs and builds/pushes to registries on default branch'
+
+inputs:
+  # Image configuration
+  image_name:
+    description: 'Base image name (e.g., myorg/myapp)'
+    required: true
+
+  dockerfile_path:
+    description: 'Path to Dockerfile'
+    required: false
+    default: './Dockerfile'
```

Comment:
Is this relative to build_context directory? I assume not.

Reply:
it is not, correct